app/models/evaluation.py

```diff
@@ -0,0 +1,100 @@
+from sqlalchemy import String, Text, Integer, DECIMAL, ForeignKey, Date, Enum
+from sqlalchemy.orm import Mapped, mapped_column, relationship
+from uuid import UUID
+from .base import Base
+from .user import User
+from enum import Enum as PyEnum
+from category import CategoryProgresses, CategoryEnum
```

Comment:
`CategoryProgresses`가 필요한가요? relationship을 위해 필요한거라면 그동안 작성된 모든 모델 코드에 다 relationship을 위한 import가 들어가야 하고 만약 필요 없다면 여기만 지우시면 됩니당

```suggestion
from .category import CategoryNameEnum
```

Reply:
이건 RecommendedChalenge 테이블에 관계설정할때  필요해서 했어요